tests/lustre/typechecking/shouldPass/test_const_prop.lus

```diff
@@ -17,7 +17,7 @@ const f = arr[fc]; --  should not inline
 const f2 = arr[1]; -- should inline to false
 const f3 = [true, true, true][2];
 
-const t3:[int, int, bool] = (t, 3, true); -- should inline to (1, 3, true)
+const t3:[int, int, bool] = {t, 3, true}; -- should inline to (1, 3, true)
 const arr3 = [1, 1+1, t1+c, 5]; -- inline all t's and simplify
 
 const p = t3.%1; -- should inline to 3
```

Comment:
Add a dummy node at the bottom of the file so that Kind 2 does not complain about:

> No main node defined in input

Reply:
I remembered you granted me permission to modify your fork. I will fix it myself.